File: Android/app/src/main/java/com/co2team/covidbuster/model/RoomCo2Data.java
package com.co2team.covidbuster.model;

import java.util.Date;

public class RoomCo2Data {

    private int co2ppm;
    private Date created;

    public RoomCo2Data(int co2ppm, Date created) {
        this.co2ppm = co2ppm;
        this.created = created;
    }
}

File: Android/app/src/main/java/com/co2team/covidbuster/model/RoomCo2Data.kt
package com.co2team.covidbuster.model;

import java.time.LocalDateTime;

public class RoomCo2Data {

    private final int co2ppm;
    private final LocalDateTime created;

    public RoomCo2Data(int co2ppm, LocalDateTime created) {
        this.co2ppm = co2ppm;
        this.created = created;
    }

    public int getCo2ppm() {
        return co2ppm;
    }

    public LocalDateTime getCreated() {
        return created;
    }
}
